pkg/controller/cloud/node_controller.go

```diff
@@ -373,7 +373,7 @@ func getNodeAddressesByProviderIDOrName(ctx context.Context, instances cloudprov
 		providerIDErr := err
 		nodeAddresses, err = instances.NodeAddresses(ctx, types.NodeName(node.Name))
 		if err != nil {
-			return nil, fmt.Errorf("NodeAddress: Error fetching by providerID: %v Error fetching by NodeName: %v", providerIDErr, err)
+			return nil, fmt.Errorf("Error fetching node by providerID: %v, and by NodeName: %v", providerIDErr, err)
```

Comment:
We should not be beginning error messages with a capital letter. Also we have labelled the second parameter as NodeName but it appears to be an error.

Reply:
In fact the first parameter is labelled as providerID but appears to also be an error.

Reply:
It's `providerIDErr`, but agreed about not starting errors with capaitals, will fix that shortly 

/hold

Reply:
the two parameters are both errors, the first for attempting to get node addresseses by provider ID, the second by node name. 

/hold cancel

Reply:
maybe make it "and error be NodeName: " just to prevent any confusion.

Reply:
done